repository/zookeeper/operator/operator.yaml

```
@@ -1,6 +1,6 @@
 name: "zookeeper"
-version: "0.1.0"
-kudoVersion: 0.3.0
+version: "0.2.0"
+kudoVersion: 0.5.0
```

could we add `appVersion` to this?   example: https://github.com/kudobuilder/operators/blob/99cc77ec52b7165f59551581c2ae400430082d54/repository/kafka/operator/operator.yaml#L5

```suggestion
kudoVersion: 0.5.0
appVersion: 3.4.14
```

---

thanks @kensipe !